.env

```diff
@@ -54,7 +54,7 @@ UBUNTU=20.04
 
 # Default versions for various dependencies
 CLANG_TOOLS=12
-CUDA=9.1
+CUDA=11.4.0
```

Comment:
It may make sense to pick the oldest supported version here instead of this so that someone doesn't accidentally use a feature introduced in a newer driver version and allow it to slip through CI.

Reply:
That would seem to be 11.0.3 for Ubuntu 20.04. Does it make sense?
(note that this variable will have to be overriden depending on the Ubuntu version selected, anyway)

Reply:
For Ubuntu 20.04 that makes sense, if we could use 18.04 I would suggest going with 10.0 instead.

I would hope that everyone is using a 11.0+ driver today though.